examples/tool_chat_template_llama.jinja

```diff
@@ -0,0 +1,110 @@
+{{- bos_token }}
+{%- if custom_tools is defined %}
+    {%- set tools = custom_tools %}
+{%- endif %}
+{%- if not tools_in_user_message is defined %}
+    {%- set tools_in_user_message = true %}
+{%- endif %}
```

Comment:
I found that for multi-turn conversations, it was better to have this set to `false` by default.